man/dockerd.8.md

```diff
@@ -148,7 +148,7 @@ following format.
   DNS search domains to use.
 
 **--exec-opt**=[]
-  Set runtime execution options. See RUNTIME EXECUTION OPTIONS.
+  Set [options for the runtime](dockerd.8.md#options-for-the-runtime).
```

Comment:
OK,  Rollback it. 
Set runtime execution options. See [options for the runtime](dockerd.8.md#options-for-the-runtime).